calico_node/startup/startup.go

```diff
@@ -779,8 +776,12 @@ func ensureDefaultConfig(c *client.Client, node *api.Node) error {
 	// for containers to speak to services running on the host (e.g. k8s
 	// pods speaking to k8s api-server, and mesos tasks registering with agent
 	// on startup).
-	if err := ensureFelixConfig(c, node.Metadata.Name, "DefaultEndpointToHostAction", "RETURN"); err != nil {
-		return err
+	//
```

Comment:
i'd added that deliberately as it happens - just to separate the two bits of info, but I can remove if it annoys you.